src/Component/SignUp/SignUp.scss

```
@@ -0,0 +1,326 @@
+// @import "./Mixin";
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.SignUp {
+  height: 100vh;
+  // background-image: url(../Login/images/loginbackground.jpg);
+  position: relative;
+  background-size: cover;
+  .main {
+    display: flex;
+    font-size: 62.5%; //10px = 1rem
+    width: 60rem;
+    height: 48rem;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    padding: 3rem 2rem 2.4rem 2rem;
+    border: 0.1rem solid black;
+    background-color: rgba(255, 255, 255, 0.9);
+    border-radius: 1rem;
+    .leftMain {
+      .nav {
+        width: 21.2rem;
+        height: 19.5rem;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        .nav-my-picture {
+          width: 14rem;
+          height: 14rem;
+        }
+        .nav-choice-picture {
+          width: 11rem;
+          height: 3.6;
+          margin-top: 2rem;
+          font-size: 1.4rem;
+          font-weight: 700;
+          background-color: white;
+          color: gray;
+        }
+      }
+    }
+    .rightMain {
+      & {
+        margin: 0 0 0 2rem;
+      }
+      .rightMain-info {
+        & {
+          margin: 0 0 2rem 0;
```

```suggestion
          margin-bottom: 2rem; 
```
이라고 작성하는게 더 깔끔해보이네요 :)

---

넵 알겠습니다.